deployments/helm/nvidia-device-plugin/templates/daemonset-gfd.yml

```diff
@@ -199,6 +205,9 @@ spec:
         - name: host-sys
           hostPath:
             path: "/sys"
+        - name: imex-nodes-config
+          hostPath:
+            path: "/etc/nvidia-imex"
```

Comment:
This needs to be relative to hostDriverRoot

Reply:
This mount is going to fail if this doesn't exist on the host.

Reply:
It will mount an empty directory

Reply:
I believe the option is something like `DirectoryOrCreate`

Reply:
https://kubernetes.io/docs/concepts/storage/volumes/#hostpath-volume-types

Reply:
@ArangoGutierrez I don't understand what the link is supposed to show. With no volume type including `Directory` the mount will be treated as a `file` if it doesn't exist and I believe it will also create the file on the host for mounting. I agree with @klueska that we probably want `DirectoryOrCreate` or at the very least `Directory` to force a user to have created the directory with the relevant config beforehand.

Reply:
This needs to be added

Reply:
Can't we unconditionally prepend `nvidiaDriverRoot`? What is the behaviour if:
* the value is `nil`
* the value is `""`
* the value is `"/"`
* the value is `"/some/path"`
* the value is `"/some/path/"`

Is there a helm function that we can use to cover these cases instead of the conditional blocks?

Reply:
` type: DirectoryOrCreate` was added in line  213.